CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # salsify_rubocop
 
+## v0.49.0
```

Comment:
Let's just bump this to `v0.48.1` -- it's unconventional, but what we've tried to do with this gem is keep the minor version in sync with rubocop.